client/public/editPodcastPage/editPodcastPage.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Add New Podcast</title>
+    <link rel="stylesheet" href="editPodcastPage.css">
+    <link rel="stylesheet" href="../css/formStyle.css">
+    <script src="./editPodcastPage.js"></script>
+</head>
+<body>
+<div class="container">
+    <div class="container-middle">
+        <div class="form-title">Edit Podcast Information</div>
+        <label class="form-labels">Title</label>
+        <input class="form-button" type="text" id="title" value="Title Placeholder">
+        <label class="form-labels">Description</label>
+        <input class="form-button" type="text" id="description" value="Description Placeholder">
```

Comment:
for a field like description I would use textarea and not input element

Reply:
Fixed